CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-# CRPropa NEXT
+# CRPropa vNext
 
 ### Bug fixes:
+* Re-added TorroidalHaloField and LogarithmicSpiralField models.
```

Comment:
Torroidal => Toroidal

Reply:
We call it _Toroidal_ in the JF12Field class (JF12Field::getToroidalHaloField) but _Torroidal_ in the GalacticMagneticField.h file (class TorroidalHaloField: public MagneticField). So if we want to use only _Toroidal_, we have to change it in GalacticMagneticField.h too. Should we initiate this change?

Reply:
I think so, @reichherzerp.
This is clearly a typo that could frustrate some users.

Reply:
Done. And also updated in GalacticMagneticField.h. Note was added in CHANGELOG to make the syntax change clear.